src/sentry/conf/server.py

```diff
@@ -1046,6 +1046,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:api-keys": False,
     # Enable multiple Apple app-store-connect sources per project.
     "organizations:app-store-connect-multiple": False,
+    # Enable auth provider configuration through api
+    "organizations:auth-provider-config": False,
```

Comment:
Can we rename this to `api-auth-provider` so it's obvious and it's next to the other API-related flags?

Reply:
renamed